lookout/style/format/code_generator.py

```diff
@@ -74,13 +74,7 @@ def apply_predicted_y(self, vnodes: Sequence[VirtualNode],
         result = []
         for vnode, y_new in self._iterate_vnodes(vnodes, vnodes_y, y_pred):
             vnode = vnode.copy()
-            # if there is no prediction (y_new) for current VirtualNode (vnode)
-            # its value is set to None.
-            if y_new == vnode.y and not vnode.value:
-                # Skipping all unmodified NOOPS and empty tokens
-                # No need to finalize
-                continue
-            else:
+            if y_new != vnode.y or vnode.value:
```

Comment:
We should not let NOOPS through because they are useless for code generation

Reply:
but it is just negation of the same condition it was before

Reply:
Oh fuck yes